crates/vision/src/field_color_detection.rs

```diff
@@ -98,6 +103,7 @@ mod test {
             blue_chromaticity_threshold: 0.38,
             green_chromaticity_threshold: 0.43,
             green_luminance_threshold: 255.0,
+            luminance_threshold: 255.0,
```

Comment:
Is this preventing to detect segments as field color in tests? Probably to not alter the tests, it should be zero? 🤔

Reply:
Done